README.md

```diff
@@ -1,2 +1,27 @@
-# hra-ui
-Human Reference Atlas UIs Monorepo
+# Human Reference Atlas (HRA) and Functional Tissue Unit Explorer (FTU) - User Interfaces
+
+[![Tests]()](https://github.com/hubmapconsortium/hra-ui/actions/workflows/ci.yml)
+[![GitHub last commit]()](https://github.com/hubmapconsortium/hra-ui/commits/develop)
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE)
+[![Quality Gate Status]()](https://sonarcloud.io/project/overview?id=hubmapconsortium_hra-ui)
+[![SonarCloud Coverage]()](https://sonarcloud.io/component_measures?metric=Coverage&id=hubmapconsortium_hra-ui)
+[![Documentation Status]()]()
+
+## Website Links
+
+- FTU User Interface (FTU)
+  - Production: <>
```

Comment:
Production (alt) is production, so remove (alt) from below and remove this empty line